apps/autoscheduler/FunctionDAG.h

```diff
@@ -348,6 +348,11 @@ struct FunctionDAG {
     // An edge is a producer-consumer relationship
     struct Edge;
 
+    struct SymbolicRange {
```

Comment:
Halide::Range is min, extent, and Halide::Interval is min, max. In keeping with that convention, how about naming this SymbolicInterval?

Reply:
Good idea, done